nanome/_internal/_plugin_instance.py

```diff
@@ -48,6 +86,12 @@ def _run(self):
                 time.sleep(sleep_time)
                 last_update = timer()
 
+                dt = last_hook_unref_check - timer()
```

Comment:
I think this subtraction is backwards? timer will have a greater value than last_check. So dt will always be negative and the if will always fail.